datafusion/core/tests/memory_limit/mod.rs

```diff
@@ -141,7 +141,7 @@ async fn join_by_expression() {
     TestCase::new()
         .with_query("select t1.* from t t1 JOIN t t2 ON t1.service != t2.service")
         .with_expected_errors(vec![
-           "Resources exhausted: Additional allocation failed with top memory consumers (across reservations) as: NestedLoopJoinLoad[0]",
+            "Resources exhausted: Additional allocation failed with top memory consumers (across reservations) as: NestedLoopJoinLoad[0]",
```

Comment:
Could you please revert all unintentional space-adding changes in the PR as well? Thanks!

Reply:
Yes definitely. I think this happened when I ran `cargo fmt` for some reason haha. I'll revert.